### DISCUSSION TITLE
spring tab showing blank with 4.0-M4 using spring boot 3.1.5 and Jdk21

### DISCUSSION THREAD

**Discussion by jmjava** [hawtio/hawtio#3179] [Q&A]
I get the other two tabs, but the springboot tab is coming up whitescreen.   Any ideas?

![image](https://github.com/hawtio/hawtio/assets/591751/f7cbfebf-4ef7-4963-b1a8-055745b22f02)

**Comment by tadayosi** [comment]
Make sure you added `spring-boot-starter-actuator` dependency. If you did, it's worth checking what are the differences between your app's properties and that from an working example.
https://github.com/hawtio/hawtio/blob/3.x/examples/springboot/src/main/resources/application.properties

**Reply by steverogers321** [reply]
Is this issue resolved? or still open?

**Comment by jmjava** [comment]
I can test again with most recent beta.  The last time I tried it didn't
work.

**Reply by steverogers321** [reply]
Can you type the link to your repo here? so that i can download and check it in my local.

**Comment by jmjava** [comment]
Unfortunately we are not open source.  Our project just has JDK 21 and
spring 3.1.5 with the actuator etc.

**Reply by tadayosi** [reply]
Can you try Hawtio 4.0-beta-1?

**Reply by jmjava** [reply]
running into different issues now with my web security config:  

mappable servlet in your servlet context: {io.hawt.web.auth.keycloak.KeycloakServlet=[/api/actuator/hawtio/keycloak/*],

trying to disable this 

KeycloakServlet appears to be directly included in beta-1 jar.  There does not appear to be a way not to load it that I can find

**Reply by tadayosi** [reply]
We don't need to disable `KeycloakServlet` even when you don't use Keycloak. It should just do nothing than staying there. What's the real issue then?
